### ISSUE TITLE
Latest activesupport version breaks test-helper

### ISSUE THREAD

**Issue by vjdhama** [lipro-cookbooks/test-helper#6]
Latest activesupport version(5.0.0) which requires ruby `version > 2.2.2` does not work with latest embedded chef ruby version(2.1.8). 

Stacktrace :
```
       Recipe: test-helper::dump_node

         * chef_gem[activesupport] action install
           
           ================================================================================
           Error executing action `install` on resource 'chef_gem[activesupport]'
           ================================================================================
           
           Mixlib::ShellOut::ShellCommandFailed
           ------------------------------------
           Expected process to exit with [0], but received '1'
           ---- Begin output of /opt/chef/embedded/bin/gem install activesupport -q --no-rdoc --no-ri -v "5.0.0" ----
           STDOUT: Successfully installed i18n-0.7.0
           Successfully installed thread_safe-0.3.5
           Successfully installed tzinfo-1.2.2
           Successfully installed minitest-5.9.0
           Successfully installed concurrent-ruby-1.0.2
           STDERR: ERROR:  Error installing activesupport:
           	activesupport requires Ruby version >= 2.2.2.
           ---- End output of /opt/chef/embedded/bin/gem install activesupport -q --no-rdoc --no-ri -v "5.0.0" ----
           Ran /opt/chef/embedded/bin/gem install activesupport -q --no-rdoc --no-ri -v "5.0.0" returned 1

**Comment by patcon** [comment]
PR pending in #7

**Comment by rexut** [comment]
PR pending in #8 (#7 will closed as long as it is part of #8)

**Comment by rexut** [comment]
Please, use the new version 1.2.0 from Chef Supermarket
